epochStart/shardchain/trigger.go

```diff
@@ -742,6 +754,7 @@ func (t *trigger) SetProcessed(header data.HeaderHandler, _ data.BodyHandler) {
 	t.mapHashHdr = make(map[string]*block.MetaBlock)
 	t.mapNonceHashes = make(map[uint64][]string)
 	t.mapEpochStartHdrs = make(map[string]*block.MetaBlock)
+	t.mapFinalizedEpochs = make(map[uint32]struct{}, 0)
```

Comment:
What about RevertStateToBlock, do we need to set this map back?

Reply:
none of these maps are set in the revert - they are repopulated.

Reply:
ok